config/current.go

```diff
@@ -0,0 +1,37 @@
+package config
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"git_applet/config/migration"
+	v1 "git_applet/config/migration/v1"
+	v2 "git_applet/config/migration/v2"
+)
+
+type (
+	Config   = v2.Config
+	Tracking = v2.Tracking
+)
+
+var LatestVersion int = 2
+
+func Load(filename string, dumpMigrations bool, logger *log.Logger) (Config, error) {
```

Comment:
config loader struct maybe to inject the logger ?

Reply:
It'll make its usage harder, I'm not sure if it's worth it for 1 variable

Reply:
It's just a matter of segregating deps (logger) to struct fields and actual function arguments

Reply:
Yeah, I got the reason, I just feel it's way too early for that. It'd be creating yet another component (`ConfigLoader`), and forcing `main.go` to instantiate it and store it, just to end up only calling a function once in the whole app lifecycle. The `config` package is already quite abstracted, more abstraction atop this feels premature